lib/test.coffee

```diff
@@ -80,6 +80,9 @@ class Test
     assert.isNull error
     assert.isNotNull response, 'Response'
 
+    # Headers
+    @response.headers = response.headers
```

Comment:
Any reason this line is different from https://github.com/cybertk/abao/pull/73/files#diff-e75dcbdd9657986e1e2d6c3d0359a6d2R92

Should we be modifying the same response object?